src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img src="/moyo_1.svg" alt="Logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">APPLE</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">SAMSUNG</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">SMARTPHONES</a></li>
+          <li data-qa="hover" class="nav__item"><a href="#" class="nav__link">LAPTOPS & COMPUTERS</a></li>
```

Comment:
move `data-qa="hover"` to `a` tag

Reply:
data-qa="hover" underline in red
![image](https://github.com/mate-academy/layout_moyo-header/assets/70914734/d32c479d-35db-4c71-8195-04a53dd57a0c)